specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2021-12-01-preview/Tables.json

```diff
@@ -0,0 +1,742 @@
+        },
+        "sourceTable": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The table to restore data from."
+        }
+      },
+      "description": "Restore parameters."
+    },
+    "ResultStatistics": {
+      "type": "object",
+      "properties": {
+        "progress": {
+          "readOnly": true,
+          "type": "number",
+          "multipleOf": 0.01,
+          "description": "Search job completion percentage."
+        },
+        "ingestedRecords": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of rows that were returned by the search job."
+        }
+      },
+      "description": "Search job execution statistics."
+    },
+    "TableProperties": {
+      "type": "object",
+      "properties": {
+        "retentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 730,
+          "description": "The data table data retention in days, between 4 and 730. Setting this property to null will default to the workspace retention."
+        },
+        "totalRetentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 2555,
+          "description": "The table data total retention in days, between 4 and 2555. Setting this property to null will default to table retention."
+        },
+        "archiveRetentionInDays": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The table data archive retention in days. Calculated as (totalRetentionInDays-retentionInDays)"
+        },
+        "searchResults": {
+          "$ref": "#/definitions/SearchResults",
+          "description": "Parameters of the search job that initiated this table."
+        },
+        "restoredLogs": {
+          "$ref": "#/definitions/RestoredLogs",
+          "description": "Parameters of the restore operation that initiated this table."
+        },
+        "resultStatistics": {
+          "$ref": "#/definitions/ResultStatistics",
+          "description": "Search job execution statistics."
+        },
+        "plan": {
+          "type": "string",
+          "description": "The table plan.",
+          "enum": [
+            "Basic",
+            "Analytics"
+          ],
+          "x-ms-enum": {
+            "name": "TablePlanEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Basic",
+                "description": "Basic - logs that are adjusted to support high volume / low value verbose logs."
+              },
+              {
+                "value": "Analytics",
+                "description": "Analytics - logs that allow monitoring and analytics."
+              }
+            ]
+          }
+        },
+        "lastPlanModifiedDate": {
```

Comment:
compare with latest stable 2021-06-01, rename property is kind of breaking change which need to breaking change review board's approval, can you confirm it is necessary change?

Reply:
It is, previous property is only supported in previous versions, not in the new one. it is a necessary change, its not a rename but a new property while the previous one was deprecated.

Reply:
if not a renaming, looks there is 3 properties removed like isTroubleshootingAllowed etc. It need breaking change review board approval
and you also need to fix some other validation failure like modalValidation and prettier check before I can merge